gettsim/visualization.py

```diff
@@ -95,25 +57,16 @@ def plot_dag(
         visualization. For the full list of options, see the tutorial about
         `visualization <../docs/tutorials/visualize.ipynb>`_. By default, all nodes are
         shown.
-    labels : bool, default True
-        Annotate nodes with labels.
-    tooltips : bool, default False
-        Experimental feature which makes the source code of the functions accessible as
-        a tooltip. Sometimes, the tooltip is not properly displayed.
-    plot_kwargs : dict
-        Additional keyword arguments passed to :class:`bokeh.models.Plot`.
-    arrow_kwargs : dict
-        Additional keyword arguments passed to :class:`bokeh.models.Arrow`. For example,
-        change the size of the head with ``{"size": 10}``.
-    edge_kwargs : dict
-        Additional keyword arguments passed to :class:`bokeh.models.MultiLine`. For
-        example, change the color with ``{"fill_color": "green"}``.
-    label_kwargs : dict
-        Additional keyword arguments passed to :class:`bokeh.models.LabelSet`. For
-        example, change the fontsize with ``{"text_font_size": "12px"}``.
-    node_kwargs : dict
-        Additional keyword arguments passed to :class:`bokeh.models.Circle`. For
-        example, change the color with ``{"fill_color": "orange"}``.
+    orientation :str,default "v"
+         Whether the graph is horizontal or vertical
+    show_labels : bool, default True
+        Whether the graph is annotated with labels.
+    show_legend: bool, default False
```

Comment:
Setting this to True did not add a legend for me. Can you include an example in the Tutorial?

Reply:
Hi Christian, we made changes accordingly to all of your comments. As for the legend argument, we deleted it in this version because we think it does not add additional value. Since all the nodes are the same color and the node names are shown anyway, either directly or as hover info.